package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "lab-express-chuck-norris",
+  "version": "1.0.0",
+  "description": "![Ironhack Logo](https://i.imgur.com/1QgrNNw.png)",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Remember to add the scripts that you are going to use, now you only have the "test" one. E.g:
```
"scripts": {
    "start": "node app.js",
    "start-dev": "nodemon --inspect app.js"
}
```